components/pickerbutton/index.css

```diff
@@ -63,7 +63,4 @@ governing permissions and limitations under the License.
 .spectrum-PickerButton-icon {
   /* don't be small, ever */
   flex-shrink: 0;
-
-  /* remove margin used for centering */
-  margin: 0 !important;
```

Comment:
This line was causing the validation icon to position incorrectly in Datepicker, as seen here:

![Screenshot 2023-01-30 at 11 32 54 AM](https://user-images.githubusercontent.com/25614178/215605664-74883921-7143-4ce4-8718-b2a98810230f.png)

Removing it allows the icon to be closer to the correct position:
![Screenshot 2023-01-30 at 12 10 47 PM](https://user-images.githubusercontent.com/25614178/215605844-db9766c6-cdf3-4dbc-8303-ce939d5b92d2.png)